includes/core/assets/js/venue-deny-list.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Remove unwanted blocks from given post type.
+ */
+
+const disableBlocks = [
+	'gatherpress/add-to-calendar',
+	'gatherpress/attendance-list',
+	'gatherpress/attendance-selector',
+	'gatherpress/event-date',
+	'gatherpress/venue',
+];
+
+wp.domReady(function () {
+	Object.keys(disableBlocks).forEach(function (key) {
+		const blockName = disableBlocks[key];
+		if (blockName && wp.blocks.getBlockType(blockName) !== undefined) {
+			wp.blocks.unregisterBlockType(blockName);
+		}
+	});
+});
```

Comment:
JS files should be added to `entry` in webpack and handled with build script.

Reply:
@mauteri can you do that for me? IE, show me how?

Reply:
added to transpiler